src/service.ts

```diff
@@ -129,6 +129,13 @@ export class DebugService implements IDebugger, IDisposable {
     Signal.clearData(this);
   }
 
+  /**
+   * Checking lifecycle state of debugger of current client
+   */
+  isLifeCycleOnKernelState(): boolean {
```

Comment:
This could be made `private`.

Reply:
Or maybe not even necessary if used only once within the service?

Reply:
I thought we will use that in another place. For now in service we only used it in one place. It's better to remove that for now